### ISSUE TITLE
Narrator always announces '0' instead of the actual value for SpinButton with suffix

### ISSUE THREAD

**Issue by wutonghao** [microsoft/fluentui#18405]
### Environment Information
- **Package version(s)**: 8.16.0, also repos in older versions
- **Browser and OS versions**: Edge

### Describe the issue:
Narrator always announces '0' instead of the actual value for SpinButton with suffix.

### Please provide a reproduction of the issue in a codepen:
https://fabricweb.z5.web.core.windows.net/oufr/8.16.0/#/examples/spinbutton
See the SpinButton example with suffix

#### Actual behavior:
There are two issues here:
1. When we select/tab to the SpinButton with suffix, the Narrator announces things in this order: control label ('With suffix') -> 'SpinButton' -> ** '0' is announced here even if the actual value is not '0'** -> min & max -> actual value + 'selected'
2. When we use 'Up' and 'Down' key to adjust the value, the Narrator only announces '0' after each adjustment

#### Expected behavior:
1. Narrator should announce the actual value instead of '0' when SpinButton with suffix is selected
2. Narrator should announce the actual value instead of '0' after each increment/decrement

**Comment by gouttierre** [comment]
@wutonghao - thanks for filing this issue with us and providing details to the problem.

@miroslavstastny /  @smhigley - Would you be able to confirm if this is a regression, or if this behavior is an issue?

**Comment by smhigley** [comment]
This is a Narrator bug; it looks like it doesn't respect the `aria-valuetext` attribute, and also for some reason has trouble with the custom string in the `value` attribute (which actually shouldn't even be used when `-valuetext` is present). I would guess that Narrator approaches spinbuttons solely by calculating the numeric value, and can't handle custom text.

We should log a bug against Narrator, but we can also implement a temporary fix with `aria-valuenow="numeric value without suffix"`. I verified that this works with Narrator -- it still won't read the suffix, but at least it reads the number.

**Comment by spmonahan** [comment]
I've just tested this with Windows 11 and Edge 100.0.1185.36 and the suffix is being read correctly.

I think it's safe to close this as fixed but defer to @gouttierre and @smhigley
